public/editor/scripts/project/project.js

```diff
@@ -244,15 +244,21 @@ define(function(require) {
               return callback(err);
             }
 
-            // Look for an HTML file to open, ideally index.html
-            var indexPos = 0;
+            // Look for an HTML file to open
```

Comment:
I guess I'll wait for #1822 to finish merging, pull in locally, rebase and then move ahead from there.